spec-examples/src/test/java/org/beanvalidation/specexamples/constraintmetadata/EmployeeImpl.java

```diff
@@ -32,7 +29,7 @@ public Roles getRoles() {
 	}
 
 	@Override
-	public Map<@NotNull String, @Valid Account> getAccounts() {
```

Comment:
Would be nice to double-check on this - they seemed redundant to me as they are already given on the overridden super-type method.